modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb

```diff
@@ -3,5 +3,235 @@
 module ClaimsApi
   class Form0966V0ControllerSwagger
     include Swagger::Blocks
+
+    swagger_path '/form/0966' do
+      operation :post do
+        key :summary, 'Accepts 0966 Intent to File form submission'
+        key :description, 'Accepts JSON payload. Full URL, including\nquery parameters.'
+        key :operationId, 'post0966itf'
+        key :tags, [
+          'Intent to File'
+        ]
+
+        parameter do
+          key :name, 'api_key'
```

Comment:
This should be `apikey` (no underscore) unless something about swagger blocks transforms it.

Reply:
Thanks for the heads up, I'll correct this